cirq/vis/heatmap.py

```diff
@@ -268,21 +271,24 @@ def _plot_on_axis(self, ax: plt.Axes, **collection_options: Any) -> mpl_collecti
         return collection
 
     def plot(
-        self, ax: Optional[plt.Axes] = None, **collection_options: Any
+        self, ax: Optional[plt.Axes] = None, **kwargs: Any
     ) -> Tuple[plt.Axes, mpl_collections.Collection]:
         """Plots the heatmap on the given Axes.
         Args:
             ax: the Axes to plot on. If not given, a new figure is created,
                 plotted on, and shown.
-            collection_options: keyword arguments passed to mcoll.PolyCollection().
+            kwargs: Updates keyword arguments passed during construction by
+                calling `self.update_config(**kwargs)` before plotting.
+                See __init__ for more details on the allowed arguments.
         Returns:
             A 2-tuple ``(ax, collection)``. ``ax`` is the `plt.Axes` that
             is plotted on. ``collection`` is the collection of paths drawn and filled.
         """
         show_plot = not ax
         if not ax:
             fig, ax = plt.subplots(figsize=(8, 8))
-        collection = self._plot_on_axis(ax, **collection_options)
+        self.update_config(**kwargs)
```

Comment:
Will this make it so that future calls to `plot` have the same settings? e.g.,

```python
heatmap.plot(vmin=0, vmax=0.1)

heatmap.plot()  # Also has vmin=0, vmax=0.1?
```



If so, should there be a "local config" each time `plot` is called?

Reply:
I think that makes sense. Updated the code to ensure that plot doesn't modify the internal config.